test/e2e/kubeproxy.go

```diff
@@ -38,7 +40,6 @@ import (
 const (
 	endpointHttpPort        = 8080
 	endpointUdpPort         = 8081
-	endpointHostPort        = 8082
```

Comment:
It had been in its own commit before squashing. Will extract it again.

About the why: it doesn't seem to be used.